Wikipedia/Code/WMFDatabaseHouseKeeper.swift

```diff
@@ -33,7 +33,7 @@ import Foundation
  
         */
         
-        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: -30, date: Date()) else {
+        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: 0 - WMFExploreFeedMaximumNumberOfDays, date: Date()) else {
```

Comment:
(minor) the name of the variable `thirtyDaysAgoMidnightUTC` is a little misleading now